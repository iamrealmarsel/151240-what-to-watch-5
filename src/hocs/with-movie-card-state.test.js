import React from "react";
import TestRenderer from "react-test-renderer";
import withMovieCardState from "./with-movie-card-state";


const MockComponent = () => {
  return <article></article>;
};

const MockComponentContainer = withMovieCardState(MockComponent);

test(`render withMovieCardState`, () => {
  const tree = TestRenderer
    .create(
        <MockComponentContainer />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
