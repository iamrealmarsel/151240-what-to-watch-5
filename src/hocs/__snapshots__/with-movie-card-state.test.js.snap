// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withMovieCardState 1`] = `<article />`;
